ivadomed/scripts/download_data.py

```diff
@@ -23,8 +23,11 @@
             `Spine Generic <https://github.com/spine-generic/data-multi-subject>`_.
             Used for Tutorial and example in Ivadomed."""},
     "data_testing": {
-        "url": ["https://github.com/ivadomed/data-testing/archive/r20210823.zip"],
+        "url": ["https://github.com/MotionCorrect/data-testing/archive/refs/tags/r20220116.zip"],
         "description": "Data Used for integration/unit test in Ivadomed."},
+    "data_multi_testing": {
+        "url": ["https://github.com/MotionCorrect/data_multi-sessions-contrasts/archive/refs/tags/v2022-01-06.zip"],
+        "description": "Large Data Used for multi-session contrasts integration/unit test in Ivadomed."},
```

Comment:
Altered the source of these two data_XXXX files used for unit testing. 
When PR is approved, I will submit PR to update those two folders.